core/src/test/java/org/eclipse/hono/connection/ConnectionFactoryImplTest.java

```diff
@@ -72,7 +72,7 @@ public void testConnectInvokesHandlerOnfailureToConnect() throws InterruptedExce
         factory.connect(null, null, null, connectionHandler);
 
         // THEN the connection attempt fails and the given handler is invoked
-        assertTrue(latch.await(200, TimeUnit.MILLISECONDS));
+        assertTrue(latch.await(2000, TimeUnit.MILLISECONDS));
```

Comment:
My machine sems to be too slow. Test fails with 200 msec on my machine. I can also omit this change, but thought maybe others also have this problem.